interface/orders/types.php

```diff
@@ -63,7 +63,7 @@
 <html>
```

Comment:
The ? tooltip here does nothing

Reply:
The tooltip in the `td` tag has a little bit issue when I mouse enter on the tooltip, the tooltip will flash.
https://www.loom.com/share/66a9c9e129b24828a00b3834c391e3ff

Reply:
.tooltip { 
    pointer-events: none;
}

Reply:
I fix the flashing issue but seems like the tooltip position still has some issue, I try to set the placement but it still looks like the same.

![image](https://user-images.githubusercontent.com/22230889/92187749-37ab6400-ee8d-11ea-95a5-c44ba355b9a7.png)

Reply:
whats the tips css position set to, relative, fixed or something else. simple way to chaeck is rt click and inspect.

Reply:
Hi @tywrenn @bradymiller  and @sjpadgett 

Seems like the tooltip in `table` `td` its position will be a move to on top of the table, I write some example, please take a look.
https://codepen.io/CliffSu/pen/WNwXJQQ?editors=1010

Reply:
hi @stu01509 try data attr `data-placement="right"` or bottom. ya may have to add a position relative or fixed.